pages/index.tsx

```diff
@@ -83,157 +83,182 @@ const Home: NextPage = () => {
       hash: "bafybeidn5ubtxclqpr55l5gocwstop5moqccgoakhclqxx3uiegdu5fofi"
     }
 
+  ]);
```

Comment:
this update shouldnt change index.tsx, if its only indentation issue latest precommit script should fix it you can pull it from main branch